airgun/entities/host.py

```diff
@@ -129,6 +130,11 @@ def read_insights_recommendations(self, entity_name):
         view = self.navigate_to(self, 'Recommendations')
         return view.table.read()
 
+    def insights_tab(self, entity_name):
```

Comment:
no need for this I can `session.host.get_details('entity-host-name', widget_names='insights')`

Reply:
User can click on recommendation count on all host page which navigates directly to insights tab.
I have added this code to test that navigation scenario.

Reply:
I have updated the pr so that I can also use `session.host.get_details('entity-host-name', widget_names='insights')`.